docs/deployment/uwsgi_cookiecutter_1__nginx.rst

```diff
@@ -0,0 +1,294 @@
+.. _uwsgi_cookiecutter_part_1:
+
+uWSGI with Cookiecutter :app:`Pyramid` App Part 1: Basic uWSGI + :app:`Nginx`
+=============================================================================
+
+``uWSGI`` is a software application for building hosting services.
+It is named after the Web Server Gateway Interface (the `WSGI <https://wsgi.readthedocs.io/en/latest/>`_ specification
+to which many Python web frameworks conform).
+
+This guide will outline broad steps that can be used to get a Cookiecutter
+:app:`Pyramid` application running under under ``uWSGI`` and Nginx.  This particular
+tutorial was developed and tested on Ubuntu 18.04, but the instructions should be
+largely the same for all systems, delta specific path information for commands
```

Comment:
"delta specific" this is a wonky term that only proficient English speakers might get. Let's use "where you may adjust specific..."